src/join-wrapper.js

```diff
@@ -0,0 +1,139 @@
+// join-wrapper.js
+
+// Implements a join wrapper as per RDF-pipeline wrapper definition
+// https://github.com/rdf-pipeline/noflo-rdf-pipeline/wiki/Wrapper-API
```

Comment:
Good. Let's add a comment: "This is an initial version that we should revisit when we have a clearer use case and design for how split and join should work."